config/locales/devise/devise.en.yml

```diff
@@ -30,6 +30,7 @@ en:
         someone_requested: Someone has requested a link to change your password. If
           it was you, you can do this through the link below.
         change: Change my password
+        then_connect: Once changed, you will be able to connect using your username "%{username}" and your chosen password
```

Comment:
I don't really like the word "connect" here, and would much rather have "sign in". Also, I'd probably expand this a little bit, so something along the lines of

> After setting a new password, you will be able to sign into diaspora* again using your username "%{username}" and your new password.